app/src/main/res/layout/activity_home.xml

```diff
@@ -1,12 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="@android:color/white"
+    android:orientation="vertical">
 
-    <FrameLayout
-        android:id="@+id/activity_main_fragment_container"
+    <android.support.v7.widget.Toolbar
```

Comment:
Why are you using a Toolbar here? We are only wanting to add and icon and a text